crates/turborepo-repository/src/package_json.rs

```diff
@@ -117,8 +209,8 @@ mod test {
     #[test_case(json!({"name": "foo", "resolutions": {"foo": "1.0.0"}}) ; "berry resolutions")]
     #[test_case(json!({"name": "foo", "pnpm": {"patchedDependencies": {"some-pkg": "./patchfile"}, "another-field": 1}}) ; "pnpm")]
     #[test_case(json!({"name": "foo", "pnpm": {"another-field": 1}}) ; "pnpm without patches")]
-    fn test_roundtrip(json: Value) {
-        let package_json: PackageJson = serde_json::from_value(json.clone()).unwrap();
+    fn test_roundtrip(json: serde_json::Value) {
```

Comment:
Can we add some more tests here so we have coverage for every field that doesn't go into `rest`? I've gotten bit already by biome and serde having different default behaviors for field name changing.